data_collection/gazette/spiders/ba_botupora.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class BaBotuporaSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "2904209"
+    name = "ba_botupora"
+    start_urls = ["http://ba.portaldatransparencia.com.br/prefeitura/botupora/"]
```

Comment:
The last edition dates from 22/Março/2021. Could you take a look at the current gazettes' website?